fluss-common/src/main/java/com/alibaba/fluss/config/ConfigOptions.java

```diff
@@ -423,6 +431,27 @@ public class ConfigOptions {
                                     + "Records are fetched in batches for consumer or follower, for one request batch, "
                                     + "the max bytes size is config by this option.");
 
+    public static final ConfigOption<Duration> LOG_FETCH_WAIT_MAX_TIME =
+            key("log.fetch.wait-max-time")
+                    .durationType()
+                    .defaultValue(Duration.ofMillis(500))
+                    .withDescription(
+                            "The maximum time to wait for enough bytes to be available for a fetch log response "
+                                    + "(including fetch log request from the follower or client). "
+                                    + "This value should always be less than the "
+                                    + "'log.replica.max-lag-time' at all times to prevent frequent shrinking of ISR for "
+                                    + "low throughput tables");
+
+    public static final ConfigOption<MemorySize> LOG_FETCH_MIN_BYTES =
+            key("log.fetch.min-bytes")
```

Comment:
It is quite confusing what the `log.fetch.*` configs belong to, the server-side log storage configuration or a client configuration? Do we need to add a `client.scanner.` prefix to them? But they are also be used in replica fetcher. Maybe we should also add a `replica.log.fetch.min-bytes` like kafka.

Reply:
Done!